samples/client/petstore/csharp/OpenAPIClient/src/Org.OpenAPITools/Model/Animal.cs

```diff
@@ -32,7 +32,6 @@ namespace Org.OpenAPITools.Model
     [JsonConverter(typeof(JsonSubtypes), "ClassName")]
     [JsonSubtypes.KnownSubType(typeof(Dog), "Dog")]
     [JsonSubtypes.KnownSubType(typeof(Cat), "Cat")]
-    [JsonSubtypes.KnownSubType(typeof(BigCat), "BigCat")]
```

Comment:
Why did this go away?

Reply:
I spent a little time looking and didn't see anything immediately. I'll ping core team to see if someone else can review more in-depth.

Reply:
I am pretty sure that tjis changed because assignment of children depends upon the absence of the discriminator object. I still need to fix that.